src/main.js

```diff
@@ -0,0 +1,47 @@
+//ui elements
+import { createMenuTemplate } from './view/menu';
+import { createFilterTemplate } from './view/filter';
+import { createPath } from './view/path';
+import { createSortTemplate } from './view/sort';
+import { createEventListTemplate } from './view/eventsList';
+import { createPointPath } from './view/pointPath';
+import { createEditPoint } from './view/editPoint';
+import { createNewPoint } from './view/newPoint';
+
+//helper functions
+import { RenderPosition, renderTemplate } from './render';
+const countPoint = 3;
+
+//main block
+const body = document.querySelector('.page-body');
+
+//render menu
+const siteMenuElement = body.querySelector('.trip-controls__navigation');
```

Comment:
```suggestion
const navigationElement = body.querySelector('.trip-controls__navigation');
```

Ну или даже просто navigation